src/pages/CommunityResourcesPage/ui/CommunityResourcesPage.tsx

```diff
@@ -58,7 +58,7 @@ const CommunityResources: React.FC = () => {
       <div className={styles.filter}>
         <div className={styles.tags}>
           {displayedTags.map((tag) => (
-            <Button
+            <button
```

Comment:
This is a reusable Button, should be with capital B and the properties should be updated as per shared/ui/Button component